authentication/understanding-and-managing-pod-security-admission.adoc

```diff
@@ -8,12 +8,18 @@ toc::[]
 
 Pod security admission is an implementation of the link:https://kubernetes.io/docs/concepts/security/pod-security-standards/[Kubernetes pod security standards]. Use pod security admission to restrict the behavior of pods.
 
+// Security context constraint synchronization with pod security standards
 include::modules/security-context-constraints-psa-synchronization.adoc[leveloffset=+1]
 
+// Controlling pod security admission synchronization
 include::modules/security-context-constraints-psa-opting.adoc[leveloffset=+1]
 
+// About pod security admission alerts
 include::modules/security-context-constraints-psa-rectifying.adoc[leveloffset=+1]
 
+// Identifying pod security violations
+include::modules/security-context-constraints-psa-alert-eval.adoc[leveloffset=+2]
```

Comment:
please merge these two modules

Reply:
/lgtm
we had an offline discussion with @bergerhoffer, we decided that we would keep these separate